models/tranquil/tranquil_markets_tokenprice.sql

```diff
@@ -0,0 +1,29 @@
+{{
+    config(
+        materialized='table',
+        unique_key="token_symbol||'-'||timestamp",
+        tags=['core', 'defi', 'amm', 'lending']
+        )
+}}
+
+
+with
+final as (  
+    select 'ONE' as token_symbol, * from tokenprice_one
```

Comment:
need to use {{ ref("tokenprice_one") }} and for all of the following
otherwise i think dbt would throw an error